service/src/main/kotlin/fi/espoo/evaka/pis/EmployeeQueries.kt

```diff
@@ -44,7 +44,21 @@ WHERE (:id::uuid IS NULL OR e.id = :id) AND md.id IS NULL
     .mapTo<Employee>()
     .asSequence()
 
+private fun Handle.searchFinanceDecisionHandlers(id: UUID? = null) = createQuery(
+    // language=SQL
+    """
+SELECT e.id, e.first_name, e.last_name, e.email, e.external_id, e.created, e.updated, e.roles
+FROM employee e
+JOIN daycare ON daycare.finance_decision_handler = e.id
+LEFT JOIN mobile_device md on e.id = md.id
```

Comment:
This part is unnecessary since mobile devices won't be picked as decision handlers anyway.